src/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { MakeChange } from '../types/MakeChange';
+
+type Props = {
+  setTodos: MakeChange,
+};
+
+export const Header: React.FC<Props> = ({
+  setTodos,
+}) => {
+  const [value, setValue] = useState('');
+  const input = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    input.current?.focus();
+  }, []);
+
+  const hendlerInput = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Typo in the function name 'hendlerInput', consider renaming it to 'handleInput'. Fix it everywhere. It can helps https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364